controllers/observability/observability_controller.go

```diff
@@ -2,17 +2,23 @@ package observability
 
```

Comment:
missing unit tests for this controller

Reply:
What should I test here? It is simple, it sets up the controller and then calls the other files that are tested independently. Should I add tests for those setup steps?

Reply:
So do a simple test. But we want to make sure that the new controller works

Reply:
done